setup.sh

```diff
@@ -232,27 +232,60 @@ setup_networking() {
 		cidr=$(echo $TINKERBELL_NETWORK | grep -Eo "\/[[:digit:]]+" | grep -v "^$" | tr -d "/")
 		case "$1" in 
 			ubuntu)
-				if [ ! -f /etc/network/interfaces ] ; then
-					echo "$ERR file /etc/network/interfaces not found"
-					exit 1
-				fi
+				v16=16.04
+				version=$(. /etc/os-release && echo "$VERSION_ID")
+				if (( $(echo "$version > $v16" | bc -l) )); then
+					echo "$INFO found Ubuntu $version, using 'netplan' for network configuration"
```

Comment:
It looks like 17.10 is the first release to use netplan by default. Maybe we should change this to `>= 17.10`?